src/lib/list/selection-list.ts

```diff
@@ -203,26 +230,38 @@ export class MatListOption extends _MatListOptionMixinBase
     '[tabIndex]': 'tabIndex',
     'class': 'mat-selection-list',
     '(focus)': 'focus()',
+    '(blur)': 'onTouched()',
     '(keydown)': '_keydown($event)',
     '[attr.aria-disabled]': 'disabled.toString()'},
   template: '<ng-content></ng-content>',
   styleUrls: ['list.css'],
   encapsulation: ViewEncapsulation.None,
+  providers: [MAT_SELECTION_LIST_VALUE_ACCESSOR],
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class MatSelectionList extends _MatSelectionListMixinBase implements FocusableOption,
-    CanDisable, CanDisableRipple, HasTabIndex, AfterContentInit {
+    CanDisable, CanDisableRipple, HasTabIndex, AfterContentInit, ControlValueAccessor {
 
   /** The FocusKeyManager which handles focus. */
   _keyManager: FocusKeyManager<MatListOption>;
 
   /** The option components contained within this selection-list. */
   @ContentChildren(MatListOption) options: QueryList<MatListOption>;
 
+  /** Emits a change event whenever the selected state of an option changes. */
+  @Output() change: EventEmitter<MatSelectionListChange> =
```

Comment:
This actually needs to be `selectionChange` rather than `change`

Looks like it got inadvertently renamed in this PR.

Reply:
I intentionally named it `change` to be consistent with the other form elements supporting the `ControlValueAccessor`. I can switch it to `selectionChange` or also add a deprecated `selectionChange` output.

Reply:
Let's go with `selectionChange` since that's what the `mat-select` now uses